react/components/ProductSummaryImage/ProductImage.js

```diff
@@ -65,15 +97,17 @@ const ProductImageContent = ({
     label: badgeText,
   })
   const withCollection = maybeCollection({ productClusters })
-  const img = <img className={imageContentClassName} src={imageUrl} alt={name} />
+  const img = (
+    <img className={imageContentClassName} src={imageUrl} alt={name} />
+  )
 
   return compose(
     withBadge(showBadge),
     withCollection(showCollections)
   )(img)
 }
 
-const ProductImage : FunctionComponent = (props) => {
+const ProductImage = props => {
```

Comment:
Any particular reason for this component's type to be removed? I saw that you're typing ProductSummaryPrice

Reply:
The reason is that this file is not typescript 😁

Reply:
hahahaha sure

Reply:
I'll remove all typed components. Someone did it and I don't know why it didn't break the whole store

Reply:
nice, it makes sense